src/snowflake/snowpark/session.py

```diff
@@ -1092,9 +1128,12 @@ def _upload_unsupported_packages(
                 native_packages,
             )
 
-            if len(native_packages) > 0 and not force_push:
+            if (
+                len(native_packages) > 0
+                and not self._custom_packages_force_upload_enabled
+            ):
                 raise ValueError(
-                    "Your code depends on native dependencies, it may not work on Snowflake! Use option `force_push` "
+                    "Your code depends on native dependencies, it may not work on Snowflake! Set Session parameter 'custom_packages_force_upload_enabled' to True "
```

Comment:
"True" should probably be monospace (code) font here.

Reply:
I see that errors aren't usually monospaced in the repo as code errors are unlikely to be rendered in code font.